docker/Dockerfile.postgres

```diff
@@ -1,4 +1,4 @@
-FROM postgres:10
+FROM postgres:10.11-alpine
```

Comment:
Why is this change required? It will break the database when upgrading from the non-alpine image. See https://github.com/omarroth/invidious/pull/687